### DISCUSSION TITLE
请问怎么删除评论？没看到操作入口

### DISCUSSION THREAD

**Discussion by ichenkaihua** [walinejs/waline#178] [Q&A] [answered]
**问题描述 | Describe the bug**

用户/自己发布了评论后，在**博主**已经登录的状态下，没看到`删除`评论的入口

![image](https://user-images.githubusercontent.com/13034364/111900320-d8015880-8a6c-11eb-8a2f-1d78896b3e38.png)

**问题网站 | Website URL**

**部署方式 | Deploy Type**
- 服务部署在哪里？ | Where your waline deploy?
    - [ ] Vercel
    - [ ] CloudBase
    - [x] Docker 
    - [ ] VPS, Virtual Host
    - [ ] Other
- 数据存储在哪里？| Where your comment data store?
    - [ ] LeanCloud(https://leancloud.app) 
    - [ ] LeanCloud(https://leancloud.cn)
    - [ ] CloudBase
    - [ ] MongoDB
    - [ ] MySQL
    - [x] SQLite
    - [ ] PostgreSQL
    - [ ] Github

**Comment by lizheming** [comment, marked as answer]
`<serverURL>/ui` 可以进入后台对评论进行管理
